run_sets/testsuite.yml

```diff
@@ -8,6 +8,7 @@
 - features/channels_add.feature
 - features/push_package.feature
 - features/create_repository.feature
+- features/systemspage.feature
```

Comment:
looking at the original https://github.com/SUSE/spacewalk-testsuite-base/blob/master/run_sets/testsuite.yml#L9